### PR TITLE
Update media docs

### PR DESCRIPTION
**Pull request by hassanelsheikha**
Hi @mjordan. As I approach the end of development for the `update_media` functionality for Workbench, I thought I would create some basic documentation for it first before making the PR for the code. That way, you can spot obvious changes you want done (if any) to the functionality and have me make them before submitting the code for review. Hopefully this is to your satisfaction. Thank you!

### CODE REVIEW

**Comment by mjordan** [comment]
Looks excellent! The only comment I have is that the docs use the "note convention `!!! note`. Would you mind changing `**NOTE**` to use that convention?

**Comment by mjordan** [comment]
I think the Markdown needs to look like this:

```
!!! note
    This functionality is currently only supported for media attached to a node.
```

Difficult to say without pushing it up using mkdocs but that's how the examples look in the mkdocs documentation, and how they look in the existing markdown files.

**Comment by hassanelsheikha** [comment]
Got it. Is that correct?

File: docs/update_media.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+input_csv: update_media.csv
+```
+
+Currently, the `update_media` routine has support for the following operations:
+- Updating files attached to media
+- Updating the set of track files attached to media
+- Updating the Media Use TIDs associated with media
+- Updating the published status of media
+- Updating any plain text fields attached to media (including custom ones!)
+
```
Comment (mjordan) [inline docs/update_media.md:25]: I think we should add a warning here to say that currently, updates replace existing field values, like 

```
!!! warning
    Currently, updating media replaces existing field values.

I can open an issue for the "append", "update", "delete" `update_mode` functionality I mentioned in my review of your PR against workbench.

**Pull request merged**